### ISSUE TITLE
Support for SQL Replication from Dataverse

### ISSUE THREAD

**Issue by paulhasell** [MicrosoftDocs/power-platform#896]
Hi, at the top of the article it notes "It is **highly encouraged** that you export your Dataverse data to Azure Synapse Analytics and/or Azure Data Lake Gen2", my question is how 'highly' and does this imply that support for SQL Replication is planned to be deprecated?

We're building a Dataverse extension application to deal with high volume/transactional income data for which SQL is our preferred platform. We need to integrate large volumes of Dataverse data to pivot the income data and we don't need the additional complexity of adding Data Lake or Synapse to the mix.

Regards
Paul Hasell
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3d51ad6f-b7a7-7abd-f1d1-1d7963dd3d49
* Version Independent ID: 4612ff88-8457-2f49-8222-4fd110fd72f6
* Content: [Replicate data to Azure SQL Database using Data Export Service - Power Platform](https://docs.microsoft.com/en-us/power-platform/admin/replicate-data-microsoft-azure-sql-database)
* Content Source: [power-platform/admin/replicate-data-microsoft-azure-sql-database.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/admin/replicate-data-microsoft-azure-sql-database.md)
* Service: **power-platform**
* Sub-service: **admin**
* GitHub Login: @sabinn-msft
* Microsoft Alias: **sabinn**

**Comment by sabinn-msft** [comment]
Yes. The recommended approach is to use Synapse link for Dataverse - https://docs.microsoft.com/en-us/powerapps/maker/data-platform/export-to-data-lake. This is where we are investing and not Data Export Service.

**Comment by paulhasell** [comment]
Hi again, I've tried taking this advice but TBH, it doesn't feel a solid solution as yet. While Synapse makes the Dataverse tables accessible for direct querying, trying to do any onward processing with them has, as yet, eluded me. I have tried proxying them into another database using external tables and using data factory to push them into custom tables, neither solution have I yet been able to make work. The issue seems to be that the authentication options currently provided in the 'client' (SQL/Factory) cannot be made to access the 'ondemand' endpoint with the appropriate credential type.

It appears therefore that the only viable solution for onward processing remains direct SQL synchronisation.
